### PR TITLE
Fix the mysql_auth position in run.sh

### PR DESCRIPTION
**Pull request by stvreumi**

### CODE REVIEW

**Comment by Billy4195** [comment]
Please use `git rebase` or `git pull --rebase` to let the merge commit disappear, Thanks. 😄

**Comment by stvreumi** [comment]
rebase finished

File: docker/run.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 sudo docker run -d --name=EB-mysql --env=MYSQL_ROOT_PASSWORD=mypassword mysql:5.7
-echo -e "EB-mysql\nroot\nmypassword\nblackboard" > ../mysql_auth.txt
```
Comment (Billy4195) [inline docker/run.sh]: I think a better solution to solve the `-e` option is to check current OS while running this script.
Ref: https://stackoverflow.com/questions/3466166/how-to-check-if-running-in-cygwin-mac-or-linux/18790824

**Comment by stvreumi** [comment]
I add the shell script to check the which machine is running.
Please help me check the syntax and whether it can run on the Linux, thanks.

**Comment by stvreumi** [comment]
Hello, @Billy4195 is it OK to merge this PR? Or is there something I need to modify?

**Pull request merged**